packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 47.5.0 - November 7, 2019
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Update Org Browser to use default org when the view is opened ([PR #1763](https://github.com/forcedotcom/salesforcedx-vscode/pull/1763))
+
```

Comment:
Just an fyi - I am just about done with a QA for the apex LSP changes that Roy demo'd last week. We are trying to get them in for tomorrow too.

Reply:
Cool, lmk when the jar is merged and I'll update the log

Reply:
Cool, lmk when the jar is merged and I'll add it to the log